### PR TITLE
Object intro page

### PR DESCRIPTION
**Pull request by bryanthowell-ts**
This is a new "overview" page to go in the "Embed ThoughtSpot" menu section somewhere. The idea is that quite often people are asked to do development work without familiarity with ThoughtSpot. This page is a simplified overview of the basics of ThoughtSpot with details that are most relevant to developers, with cross-linking to the main documentation for the actual deep dive details.

I didn't add to menu, want to leave that up to Shashi

### CODE REVIEW

**Review by ashubham — APPROVED** [review]

**Comment by bryanthowell-ts** [comment]
Still need this added to the main menu before it gets merged in, wanted to leave placement and naming up to @ShashiSubramanya

**Review by ShashiSubramanya — COMMENTED** [review]
Added comments

File: docs/src/asciidocs/intro-thoughtspot-objects.adoc [hunk 1]
```diff
@@ -0,0 +1,89 @@
...
+
+Each time you pin from Search Data, it will create a new separate visualization on the Liveboard.
+
+To edit a *visualization* on a Liveboard, go to the Liveboard then click the *More (...)* menu in the top left of a visualization and choose *Edit*.
+
+*Visualizations* on a Liveboard have their own GUIDs, but the visualization objects do not exist separately from the Liveboard object.
+
+== Access control (sharing)
+ThoughtSpot's link:https://docs.thoughtspot.com/cloud/latest/data-security[access control model, window=_blank] works by an *author* or *admin* *sharing* objects to ThoughtSpot *link:https://docs.thoughtspot.com/cloud/latest/groups-privileges[groups, window=_blank]* or other indvidual users: 
+
```
[the remark this answers is not in the document]
Comment (bryanthowell-ts) [reply docs/src/asciidocs/intro-thoughtspot-objects.adoc:51]: Got this one in latest version

File: docs/src/asciidocs/intro-thoughtspot-objects.adoc [hunk 2]
```diff
@@ -0,0 +1,89 @@
...
+*link:https://docs.thoughtspot.com/cloud/latest/tags[Tags, window=_blank]* can be added to objects for organization and searching. 
+
+Tags exist as separate objects that are defined, visible, and assignable by every user.
+
+Tag search is always an *OR* operation where allowed: filtering on multiple tags results in all objects with any of the tags, not just those with all of the specified tags. 
+
+== API properties
+
+* `id`: GUID of the object. Unique within a given ThoughtSpot instance
+* `author`: GUID of user who created / uploaded the object, or had the object transfered to them
```
[the remark this answers is not in the document]
Comment (bryanthowell-ts) [reply docs/src/asciidocs/intro-thoughtspot-objects.adoc]: Got this one too

**Comment by bryanthowell-ts** [comment]
@ShashiSubramanya I merged in the parts from your intro page that weren't in this page. I think it's ready to merge in and add to the menu (at least as a useful V1)

**Comment by ShashiSubramanya** [comment]
> @ShashiSubramanya I merged in the parts from your intro page that weren't in this page. I think it's ready to merge in and add to the menu (at least as a useful V1)

Looks fine. Added the new article to nav.adoc with a few minor (editorial) corrections. Please check the ThoughtSpot objects article [here](https://visual-embed-sdk-git-object-intro-page-thoughtspot-site.vercel.app/docs/?pageid=thoughtspot-objects).

**Pull request merged**